Dockerfile.build

```diff
@@ -0,0 +1,20 @@
+ARG DISTRO='quay.io/centos/centos:stream8'
+FROM $DISTRO
+
+ARG JENKINS_HOME
+
+ARG JENKINS_USER=jenkins
+ARG JENKINS_USER_ID=1110
+ARG JENKINS_USER_GID=100
+
+
+SHELL ["bash", "-x", "-c"]
+
+COPY ./ ./
```

Comment:
This line is going to stash the entire ceph source tree at the root of the container. It's probably fine to begin with, but it might be nice to put it in a particular dir and maybe only copy only the files necessary to set up the environment?

Reply:
With the `.dockerignore` defined in this PR actually this is just a few files. The repo files are actually mounted in the container run stage.

Reply:
Gotcha, I missed that, thanks!